lochness/__init__.py

```diff
@@ -43,6 +43,48 @@
 ])
 
 
+class Subject(object):
+    def __init__(self, active, phoenix_study, phoenix_id, consent, beiwe,
+                 icognition, saliva, xnat, redcap, dropbox,
+                 box, mediaflux, mindlamp, daris, rpms,
+                 general, protected, metadata_file):
+        self.active = active
+        self.study = phoenix_study
+        self.id = phoenix_id
+        self.consent = consent
+        self.beiwe = beiwe
+        self.icognition = icognition
+        self.saliva = saliva
+        self.xnat = xnat
+        self.redcap = redcap
+        self.dropbox = dropbox
+        self.box = box
+        self.mediaflux = mediaflux
+        self.mindlamp = mindlamp
+        self.daris = daris
+        self.rpms = rpms
+        self.general_folder = general
+        self.protected_folder = protected
+        self.metadata_csv = metadata_file
+
+    def asdict(self):
```

Comment:
Previously the `Subject` object was python collection object. Here `asdict` function is eumulating `collection._asdict`, which is being used in the logging.